app/components/widget-adapter/component.js

```diff
@@ -11,7 +11,8 @@ const helpText = {
     'Tags': 'Keywords, subjects, and topics that describe the research output.',
     'Funders': 'Organizations, institutions, foundations, or groups that provided financial support for the research',
     'Dates': 'Date information about the resource was last updated by Source',
-    'Top Contributors': 'Contributors are individuals, organizations, or institutions involved in the production of the resource. Their contribution could be intellectual, material, or financial. Those listed here are the most frequently named contributors in content aggregated by SHARE.'
+    'Top Contributors': 'Contributors are individuals, organizations, or institutions involved in the production of the resource. Their contribution could be intellectual, material, or financial. Those listed here are the most frequently named contributors in content aggregated by SHARE.',
+    'filter-plaques': 'A list of your current search terms'
```

Comment:
Minor quibble, but it would look good to standardize whether there are periods or not on these!

Reply:
good call